pkg/tests/tasks/traffic/tproxy_test.go

```diff
@@ -0,0 +1,54 @@
+package traffic
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/tests/ossm"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/curl"
+	"github.com/maistra/maistra-test-tool/pkg/util/env"
+	"github.com/maistra/maistra-test-tool/pkg/util/istio"
+	"github.com/maistra/maistra-test-tool/pkg/util/ns"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+	"github.com/maistra/maistra-test-tool/pkg/util/version"
+)
+
+func TestTproxy(t *testing.T) {
+	NewTest(t).Id("T7").Groups(Full, InterOp, ARM).Run(func(t TestHelper) {
```

Comment:
`Id("T7")` is already used by another test. Change it to something else, or remove it ( I don't know if we need Id, some test don't have it set)

Reply:
I removed this ID